server/app/views/ViewUtils.java

```diff
@@ -124,4 +131,30 @@ public static PTag makeBadge(BadgeStatus status, String... extraClasses) {
             Icons.svg(Icons.NOISE_CONTROL_OFF).withClasses(Styles.INLINE_BLOCK, Styles.ML_3_5),
             span(badgeText).withClass(Styles.MR_4));
   }
+
+  /**
+   * Renders "Edited on YYYY/MM/DD" text for given instant. Provides responsive text that shows only
+   * date on narrow screens and both date and time on wider screens.
+   *
+   * @param prefix Text to use before the rendered date. Examples: "Edited on " or "Published on ".
+   * @param time Time to render. If time is missing then "unkown" will be rendered.
+   * @return Tag containing rendered time.
+   */
+  public PTag renderEditOnText(String prefix, Optional<Instant> time) {
+    String formattedUpdateTime = time.map(dateConverter::renderDateTime).orElse("unknown");
+    String formattedUpdateDate = time.map(dateConverter::renderDate).orElse("unknown");
+    return p().with(
+            span(prefix),
+            span(formattedUpdateTime)
+                .withClasses(
+                    ReferenceClasses.DATE_FIELD_BROWSER_TEST,
```

Comment:
Up to you, but we tend to just name the reference class const the same as the class itself and don't distinguish as to why it's present in the const name.

Reply:
Done.